test/functional/apps/visualize/index.ts

```diff
@@ -67,11 +67,15 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       this.tags('ciGroup9');
 
       loadTestFile(require.resolve('./_embedding_chart'));
-      loadTestFile(require.resolve('./_chart_types'));
       loadTestFile(require.resolve('./_area_chart'));
       loadTestFile(require.resolve('./_data_table'));
       loadTestFile(require.resolve('./_data_table_nontimeindex'));
       loadTestFile(require.resolve('./_data_table_notimeindex_filters'));
+
+      // this check is not needed when the CI doesn't run anymore for the OSS
+      if (!isOss) {
```

Comment:
don't we need to split tests inside the `test/functional/apps/visualize/_chart_types.ts` by **isOss** for basic chart types and **!isOss** for others? 
Just a concern for a safety, who knows when the CI will be changed..

Reply:
We decided to remove the OSS tests for this, in order to be proactive. The CI is going to change soonish I hope 🙂